src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMismatchAnalyzer.cs

```diff
@@ -0,0 +1,166 @@
+﻿#nullable enable
+
+using System.Collections.Immutable;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Linq;
+using Acuminator.Utilities.DiagnosticSuppression;
+using Acuminator.Utilities.Roslyn.Semantic.PXGraph;
+using Acuminator.Utilities.Common;
+using System.Collections.Generic;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class PXOverrideMismatchAnalyzer : PXGraphAggregatedAnalyzerBase
+	{
+		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+			ImmutableArray.Create(Descriptors.PX1096_PXOverrideMustMatchSignature);
+
+		public override bool ShouldAnalyze(PXContext pxContext, PXGraphSemanticModel graph) =>
+			base.ShouldAnalyze(pxContext, graph) && graph.Type == GraphType.PXGraphExtension;
+
+		public override void Analyze(SymbolAnalysisContext context, PXContext pxContext, PXGraphSemanticModel pxGraphExtension)
+		{
+			context.CancellationToken.ThrowIfCancellationRequested();
+
+			var allMembers = pxGraphExtension.Symbol.GetMembers();
+
+			if (!allMembers.IsDefaultOrEmpty)
+			{
+				var methodsWithPxOverrideAttribute = allMembers
+					.OfType<IMethodSymbol>()
+					.Where(m => !m.IsStatic && !m.IsGenericMethod && m.HasAttribute(pxContext.AttributeTypes.PXOverrideAttribute, false))
+					.ToList();
+
+				if (methodsWithPxOverrideAttribute.Any())
+				{
+					var allBaseTypes = pxGraphExtension.Symbol
+						.GetGraphExtensionWithBaseExtensions(pxContext, SortDirection.Ascending, true)
```

Comment:
Please use named parameters with literal values such as `true` for readability

Reply:
Done.